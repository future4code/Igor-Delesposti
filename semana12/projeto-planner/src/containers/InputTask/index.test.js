import { shallow } from 'enzyme'
import React from 'react'
import { Container, Select } from './index'
import renderer from 'react-test-renderer'




test("Testando styled do Container",()=>{
    //preparacao e execucao

     const newSnapShot = renderer.create(<Container />).toJSON()

    //verificacao
    expect(newSnapShot).toMatchSnapshot();
})

test("Testando o styled do Select", ()=>{
    //preparacao e execucao
    const newSnapShot = renderer.create(<Select />).toJSON()
    //verificacao

    expect(newSnapShot).toMatchSnapshot();
})





