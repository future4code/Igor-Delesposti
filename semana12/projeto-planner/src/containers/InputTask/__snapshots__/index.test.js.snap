// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testando o styled do Select 1`] = `
<select
  className="sc-bwzfXH gFkpqv"
/>
`;

exports[`Testando styled do Container 1`] = `
<div
  className="sc-bdVaJa ktPOvl"
/>
`;
